apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/utils/ConfigChangeContentBuilder.java

```diff
@@ -84,4 +82,19 @@ Item cloneItem(Item source) {
     return target;
   }
 
+  public ConfigChangeContentBuilder convertJsonString(String content) {
+    return gson.fromJson(content, this.getClass());
+  }
```

Comment:
could be static and don't need to create instances when invoke

Reply:
good catch.